pennylane/operation.py

```diff
@@ -182,7 +182,18 @@ def expand_matrix(base_matrix, wires, wire_order):
     torch.Tensor
     >>> res.requires_grad
     True
+
+    >>> print(expand_matrix(base_matrix, wires=[0, 2]))
+    [[ 1  2  3  4]
+     [ 5  6  7  8]
+     [ 9 10 11 12]
+     [13 14 15 16]]
+
     """
+
+    if wire_order is None:
```

Comment:
```suggestion
    if (wire_order is None) or (wire_order == wires):
```

Reply:
I was wondering if I should add the other condition as well since I saw that the docstring (doctest on line [150](https://github.com/PennyLaneAI/pennylane/blob/207f3cc3cb2e09cd8866b4c87e6181c9b557be55/pennylane/operation.py#L150) ) had this case covered or maybe I am missing something? I will add it anyways as I think this is what the ticket wanted originally.